src/server.jsx

```diff
@@ -48,6 +48,7 @@ export default locals => {
           { Object.keys(assets).filter(asset => /\.css$/.test(asset)).map(path => (
             <link key={ path } rel="stylesheet" href={ `/${path}` } />
           ))}
+          <link rel="manifest" href="/manifest.json" />
```

Comment:
Will this be able to read manifest in dev as well? 🤔

Reply:
No this file is only used for prod builds. On dev we are using HTML plugin for the HTML boilerplate.

Reply:
Sounds good! 💯